duckduckgo/iOS
# Update Delegate Signature

Task/Issue URL: https://app.asana.com/0/1204099484721401/1202285063304466/f
BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/477

**Description**:
Updates the SecureVaultDelegate signature, so we can individually turn off autofill for cards, logins and identities on macOS (and on iOS later down the road).



**Steps to test this PR**:
1.  Add a few logins
2. Disable "Save and Autofill Logins"
3. Confirm autofill is disabled in the app



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
